tools/PI/DevHome.PI/Helpers/ExpanderBehavior.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.PI.Models;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Xaml.Interactivity;
+
+namespace DevHome.PI.Helpers;
+
+public class ExpanderBehavior : Behavior<Expander>
```

Comment:
>ExpanderBehavior [](http://example.com/codeflow?start=13&length=16)

Can we add this to be in DevHome Common? Thinking others can also utilize this class in DevHome.